openslides/assignments/static/templates/assignments/assignment-detail.html

```diff
@@ -68,6 +68,15 @@ <h3 translate>Phase</h3>
           </div>
       </div>
     </div>
+    <div class="row">
+      <div class="col-md-6">
+        <!-- Tags -->
```

Comment:
Please move the whole tags block to line 54 (below 'posts' in the first column of col-md-6).